ui/main/src/app/modules/cards/components/detail/detail.component.ts

```diff
@@ -347,8 +347,16 @@ export class DetailComponent implements OnChanges, OnInit, OnDestroy, AfterViewC
             if (entityIdsRequiredToRespondAndAllowedToSendCards.length > 0) listEntitiesToRespondForHeader = this.createEntityHeaderFromList(entityIdsRequiredToRespondAndAllowedToSendCards);
             else listEntitiesToRespondForHeader = this.createEntityHeaderFromList(this.entityIdsAllowedOrRequiredToRespondAndAllowedToSendCards);
 
-            this.listVisibleEntitiesToRespond = listEntitiesToRespondForHeader.length > maxVisibleEntitiesForCardHeader ? listEntitiesToRespondForHeader.slice(0, maxVisibleEntitiesForCardHeader) : listEntitiesToRespondForHeader;
-            this.listDropdownEntitiesToRespond = listEntitiesToRespondForHeader.length > maxVisibleEntitiesForCardHeader ? listEntitiesToRespondForHeader.slice(maxVisibleEntitiesForCardHeader) : [];
+            this.listVisibleEntitiesToRespond = listEntitiesToRespondForHeader.length > maxVisibleEntitiesForCardHeader ?
+                                                    listEntitiesToRespondForHeader.slice(0, maxVisibleEntitiesForCardHeader) :
+                                                    listEntitiesToRespondForHeader;
+
+            this.listDropdownEntitiesToRespond = listEntitiesToRespondForHeader.length > maxVisibleEntitiesForCardHeader ?
+                                                    listEntitiesToRespondForHeader.slice(maxVisibleEntitiesForCardHeader) :
+                                                    [];
+
+            this.listVisibleEntitiesToRespond.sort((a, b) => a.name?.localeCompare(b.name));
+            this.listDropdownEntitiesToRespond.sort((a, b) => a.name?.localeCompare(b.name));
```

Comment:
Maybe we should sort the 'listEntitiesToRespondForHeader' list before splitting in visible an dropdown lists.

Reply:
You're right. Done. Thank you